chromium/background-scripts/background.js

```diff
@@ -327,9 +336,9 @@ function onBeforeRequest(details) {
     (uri.protocol === 'http:' || uri.protocol === 'ftp:') &&
     uri.hostname.slice(-6) !== '.onion' &&
     uri.hostname !== 'localhost' &&
-    !/^127(\.[0-9]{1,3}){3}$/.test(uri.hostname) &&
     uri.hostname !== '0.0.0.0' &&
```

Comment:
@zoracon Should this be moved into `ip_utils`?

Reply:
Go ahead and move that into `ip_utils`